components/ratgdo/cover/ratgdo_cover.cpp

```diff
@@ -26,12 +26,15 @@ namespace ratgdo {
             break;
         case DoorState::DOOR_STATE_OPENING:
             this->current_operation = COVER_OPERATION_OPENING;
+            this->position = 0.5;
             break;
         case DoorState::DOOR_STATE_CLOSING:
             this->current_operation = COVER_OPERATION_CLOSING;
+            this->position = 0.5;
             break;
         case DoorState::DOOR_STATE_STOPPED:
-            this->position = COVER_OPEN;
+            this->current_operation = COVER_OPERATION_IDLE;
+            this->position = 0.5;
```

Comment:
This is a bit of an assumption but I think its the best we can do without knowing the timing of the garage open/close

Reply:
Yes, what I would have liked is a 3 state cover type (OPEN/CLOSED/PARTIALLY OPEN). Since that's not available, I'm essentially assuming PARTIALLY OPEN is `position==0.5`.